json-logic/logic.md

```diff
@@ -0,0 +1,23 @@
+# Mapping generic logic to JSON-Logic
+
+This files summarizes how logic operations can be mapped to JSON-Logic.
+
+## Programm Control
+- `IF <condition> THEN <expression1> ELSE <expression2>`: `{"if": [<condition>, <expression1>, <expression2>]}`
+
+## Logical operations
+- `<questionId> == <value>`: `{"==", [{"var": <questionId>.value}, <value>]}`
+- `<questionId1> == <questionId2>`: `{"==", [{"var": <questionId1>.value}, {"var": <questionId2>.value}]}`
+- `<condition1> OR <condition2>`: `{"and": [<condition1>, <condition2>]}`
```

Comment:
I guess, this should be `<condition1> AND <condition2>` for the first part or `{"or": [<condition1>, <condition2>]}` for the second part so both parts match?